src/components/DefaultSectionContainer/DefaultSectionContainer.tsx

```diff
@@ -1,24 +1,33 @@
 import type { StyleProps } from '@chakra-ui/react';
 import React from 'react';
 import { Box } from '@chakra-ui/react';
+import convertToKebabCase from '../../utils/convertToKebabCase';
 
 export interface DefaultSectionContainerProps extends StyleProps {
   children: JSX.Element | JSX.Element[];
+  title?: string;
 }
 
 export const DefaultSectionContainer: React.FC<DefaultSectionContainerProps> = ({
   children,
   backgroundColor,
+  title,
   ...props
-}: DefaultSectionContainerProps) => (
-  <Box
-    position="relative"
-    paddingY={backgroundColor ? ['12', null, '28'] : 'unset'}
-    marginY={!backgroundColor ? ['8', null, '28'] : 'unset'}
-    overflow={backgroundColor ? 'hidden' : 'unset'}
-    backgroundColor={backgroundColor}
-    {...props}
-  >
-    {children}
-  </Box>
-);
+}: DefaultSectionContainerProps) => {
+  const kebabCaseTitle = title ? convertToKebabCase(title) : '';
+
+  return (
+    <Box
+      data-testid="container"
+      position="relative"
+      paddingY={backgroundColor ? ['12', null, '28'] : 'unset'}
+      marginY={!backgroundColor ? ['8', null, '28'] : 'unset'}
+      overflow={backgroundColor ? 'hidden' : 'unset'}
+      backgroundColor={backgroundColor}
+      id={kebabCaseTitle}
+      {...props}
```

Comment:
```suggestion
  return (
    <Box
      data-testid="container"
      position="relative"
      paddingY={backgroundColor ? ['12', null, '28'] : 'unset'}
      marginY={!backgroundColor ? ['8', null, '28'] : 'unset'}
      overflow={backgroundColor ? 'hidden' : 'unset'}
      backgroundColor={backgroundColor}
      id={title && convertToKebabCase(title)}
      {...props}
```

What do you think about that?

Reply:
You are right